tools/quicktype-wrapper/README.md

```diff
@@ -2,16 +2,37 @@
 
 This node module that wraps the `quicktype` tool to generate code using JSON Schema.
 
-## Build and Run
+## Install
+
+This tool can be installed as a global CLI or run as a Node script. Requires Node 8+.
+
+### CLI (Recommended)
+
+Install the CLI globally:
 
 ```sh
-npm run build
-npm start
+sudo npm run cli
```

Comment:
General practice is to never use sudo to install an npm packge. Why is this necessary?

Reply:
This is necessary for some folks because you're installing a bin `qt` globally. Otherwise you'll see:

```
Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
# or
Error: EACCES: permission denied, unlink '/usr/local/bin/qt'
```

Reply:
Removed it fyi.